documentation/slate/source/includes/objects.md

```diff
@@ -264,6 +264,10 @@ A container containing either a [admin](#admin), [poi](#poi), [address](#address
 
 ### Trip
 
+A trip corresponds to a scheduled train circulation (and all its linked real-time informations).
+
+An example : the train doing Paris-Marseille every day at 19h30 is the Trip named 456789.
+
```

Comment:
It is a bit too "sncf" view. Maybe something like:

A trip corresponds to a scheduled vehicle circulation (and all its linked real-time and disrupted routes).
An example : a train, routing a Paris to Lyon itinerary every day at 06h29, is the "Trip" named "6641".